internal/hmyapi/blockchain.go

```diff
@@ -300,19 +299,19 @@ func (s *PublicBlockChainAPI) GetLeader(ctx context.Context) string {
 }
 
 // GetValidatorSelfDelegation returns validator stake.
-func (s *PublicBlockChainAPI) GetValidatorSelfDelegation(ctx context.Context, address string) hexutil.Uint64 {
-	return hexutil.Uint64(s.b.GetValidatorSelfDelegation(internal_common.ParseAddr(address)).Uint64())
+func (s *PublicBlockChainAPI) GetValidatorSelfDelegation(ctx context.Context, address string) uint64 {
```

Comment:
For all token related value, we should use big.Int. uint64 is not enough for represent token more than 18 ONE. which is 18000000000000000000